axelor-account/src/main/java/com/axelor/apps/account/web/AccountController.java

```diff
@@ -265,4 +266,17 @@ public void verifyTemplateValues(ActionRequest request, ActionResponse response)
       TraceBackService.trace(response, e, ResponseMessageType.ERROR);
     }
   }
+
+  public void setAmountRemainingReconciliableMoveLines(
+      ActionRequest request, ActionResponse response) {
+    Long accountId = Long.valueOf(request.getContext().get("_id").toString());
```

Comment:
You won't need to do that anymore, just check my last comment in Account.xml file, you will be able to do:
Account account = request.getContext().asType(Account.class);

I don't know how it's working because I don't see where you set the "_id" in the context, but anyway :)

Reply:
Solved